code/modules/clothing/spacesuits/rig.dm

```diff
@@ -475,3 +475,33 @@
 	item_state = "rig0-t51b"
 	armor = list(melee = 35, bullet = 35, laser = 40, energy = 40, bomb = 80, bio = 100, rad = 100)
 	_color="t51b"
+
+
+//Ghetto space suit
+/obj/item/clothing/head/helmet/space/rig/ghettorig
+	name = "jury-rigged space-proof fire helmet"
+	desc = "A firefighter helmet and gas mask combined and jury-rigged into being 'space-proof' somehow."
+	icon_state = "ghettorig"
```

Comment:
Trying to do just that seems to just fuck up everything. I dunno man, this works, anything else I tried didn't.